internal/protoplugin/protoplugin.go

```diff
@@ -159,6 +159,16 @@ type Service struct {
 	Methods []*Method
 }
 
+// FQSN returns a fully qualified service name of this service.
```

Comment:
I'm adding a general comment in general at the top of the file. Note this is how this is already being done in the `protoplugin` package, which is pretty much derived from https://github.com/grpc-ecosystem/grpc-gateway/tree/master/protoc-gen-grpc-gateway/generator.

See the commentary on this subject on https://github.com/yarpc/yarpc-go/pull/797 and https://github.com/yarpc/yarpc-go/pull/623.